dkc/core/tests/test_quotas.py

```diff
@@ -0,0 +1,9 @@
+from django.apps import apps
+import pytest
+
+
+@pytest.mark.django_db
+def test_user_creation_creates_quota(user):
+    default = apps.get_app_config('core').DEFAULT_USER_QUOTA
```

Comment:
Note, if we make this a setting, we can use pytest-django's `setting` fixture to temporarily alter this value for testing, if desired.